processor/app.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask
+from flask_graphql import GraphQLView
+from app.schema import schema
+
+app = Flask(__name__)
+
+app.add_url_rule(
+    '/graphql',
```

Comment:
@RoySegall I Found the problem with the schema stitching!

Our API Gateway is trying to reach the processor service graphql server at `/` instead of `/graphql`

I've change this it `/` and it works 🎉 
```suggestion
    '/',
```